### PR TITLE
Add ClientProperties to DB

### PR DESCRIPTION
**Pull request by BrennanConroy**
Already updated the DBs

### CODE REVIEW

File: src/BenchmarksDriver/Serializers/SignalRSerializer.cs [hunk 1]
```diff
@@ -195,6 +198,7 @@ class SignalRSerializer : IResultsSerializer
                     p.AddWithValue("@WebHost", serverJob.WebHost.ToString());
                     p.AddWithValue("@Transport", clientJob.ClientProperties["TransportType"]);
                     p.AddWithValue("@HubProtocol", clientJob.ClientProperties["HubProtocol"]);
+                    p.AddWithValue("@ClientProperties", string.Join(',', clientJob.ClientProperties.ToArray()));
```
Comment (sebastienros) [inline src/BenchmarksDriver/Serializers/SignalRSerializer.cs]: That, or a json serialized array in case you need to process it programatically ?

Comment (sebastienros) [reply src/BenchmarksDriver/Serializers/SignalRSerializer.cs]: Or a json document, with property names.

File: src/BenchmarksDriver/Serializers/SignalRSerializer.cs [hunk 2]
```diff
@@ -39,6 +39,7 @@ class SignalRSerializer : IResultsSerializer
                         [WebHost] [nvarchar](50) NOT NULL,
                         [Transport] [nvarchar](50) NOT NULL,
                         [HubProtocol] [nvarchar](50) NOT NULL,
+                        [ClientProperties] [nvarchar](200) NOT NULL,
```
Comment (sebastienros) [inline src/BenchmarksDriver/Serializers/SignalRSerializer.cs]: Not big enough, you'll get properties that are bigger than this. Try 2K.

File: src/BenchmarksDriver/Serializers/SignalRSerializer.cs (hunk 1 above)
Comment (BrennanConroy) [reply src/BenchmarksDriver/Serializers/SignalRSerializer.cs]: `string.Split` not good enough?

Comment (sebastienros) [reply src/BenchmarksDriver/Serializers/SignalRSerializer.cs]: There is no escaping, so no guaranty you can deserialize it with a split.

**Comment by BrennanConroy** [comment]
🆙 📅

**Pull request merged**